rpm/tarantool-checks.spec

```diff
@@ -22,10 +22,13 @@ Easy, terse, readable and fast function arguments type checking.
 mkdir -p %{br_luapkgdir}
```

Comment:
Shouldn't we do `mkdir -p %{br_luapkgdir}/override` as well?

Reply:
Added